packages/moonstone/Marquee/Marquee.js

```diff
@@ -1,6 +1,9 @@
 /**
  * Provides components for displaying and controlling marqueed text.
  *
+ * @example
+ * <MarqueeText>Lorem ipsum dolor sit amet</MarqueeText>
```

Comment:
There's no `MarqueeText` component anymore.  Can you also make a PR for docs that removes the import?  Sadly, this breaks the docs completely for this page.  I was hoping it'd be caught within the live editor.